### DISCUSSION TITLE
After upgrade to 3.7.6, error accessing  IPAM -> Prefixes

### DISCUSSION THREAD

**Discussion by anders-nc** [netbox-community/netbox#15848] [Help Wanted!] [answered]
After the upgrade to 3.7.6 ( from 3.7.5)  i now see that i cant access Prefixes
We get an error

Server Error
There was a problem with your request. Please contact an administrator.
The complete exception is provided below:
<class 'AttributeError'>
'NoneType' object has no attribute 'objects'
Python version: 3.11.2
NetBox version: 3.7.6
Plugins: 
  netbox_bgp: 0.12.1
  netbox_documents: 0.6.4
  netbox_otp_plugin: 1.0.7
  netbox_secrets: 1.10.2
  netbox_topology_views: 3.9.1

So when going to https://netbox.domain.com/ipam/prefixes or clicking IPAM->Prefixes i get the error
However accessing a specific prefix seems to work i.e. like https://netbox.domain.com/ipam/prefixes/100

**Comment by candlerb** [comment, marked as answer]
I would start by disabling all your plugins and see if that fixes the problem; if it does then re-enable one by one to find the problematic one.

If it doesn't, then please post a full backtrace (set DEBUG=True in configuration.py)

Do you have any custom fields defined on Prefix? Have you changed the default table layout?

**Reply by anders-nc** [reply]
I found out that i it was the dns plugin and custom fields, we removed dns plugin since it failed during update.
Thanks for the guidance
